azure/7.0/ha-port1-mgmt/README.md

```diff
@@ -0,0 +1,69 @@
+# Deployment of a FortiGate-VM (BYOL/PAYG) Cluster on the Azure
```

Comment:
Is is necessary to make a copy in a new folder for each fortigate version? Or can they just be branches (with instructions in the readme)


Seems like a lot of extra effort to review and maintain multiple versions this way.

Maybe a future version could get rid of all the sub directories?

Depends on how incompatible the 7.x version is compared to the 6.4 i guess

Reply:
7.x will have some different topology in comparing to 6.x.  hence, that's why the new folder for now.